vast/integration/vast_integration_suite.yaml

```diff
@@ -580,6 +580,78 @@ tests:
       - command: --config=@./vast-pipelines.yaml export json '#type == "suricata.dns"'
       - command: --config=@./vast-pipelines.yaml export json '#type == "suricata.fileinfo"'
 
+  Pipeline operator parsing everything but summarize:
```

Comment:
Please also add a few tests that start with an expression rather than directly starting with a pipeline.

Reply:
I added more integration tests in https://github.com/tenzir/vast/pull/2877/commits/ec668124f7bba019aac36891109e710c38db7f42  and https://github.com/tenzir/vast/pull/2877/commits/aef765100effc8f20e6587da728077c1aef27a8f